src/org/zaproxy/zap/extension/ascanrules/CommandInjectionPlugin.java

```diff
@@ -623,5 +660,34 @@ private static double getResponseTimeAverage(List<Long> responseTimes) {
 
         return result / responseTimes.size();
     }
+	
+    /*Generate payload variants for uninitialized variable waf bypass
+     *https://www.secjuice.com/web-application-firewall-waf-evasion/
+     *
+     * @param cmd the cmd to insert uninitialized variable
+     */
+    private static String insertUninitVar(String cmd){
+        //get a random 1-10 lowercase letters long variable name
+        int randLength = ThreadLocalRandom.current().nextInt(1,11);

```

Comment:
For my 2 cents going to 11 especially if it's inserted multiple times is likely to knock up against length/size constraints. I'd suggest just going `nextint(1,2)` here.

Reply:
I think you mean nextint(1,3), nextint(1,2) is 1.